index.html

```diff
@@ -707,7 +710,7 @@ <h3>Validate Claim</h3>
           announcements, and correspondence.
           <br><br>
           These business rules are entirely the province of the verifier. Only after
-          satisfying their rules should a verifier rely upon a claim as an
+          satisfying their rules should a verifier rely upon claims within a credential as an
```

Comment:
```suggestion
          satisfying their rules should a verifier rely upon a claim as an
```
The mention of the claim in the credential is, IMO, obvious. We don't verify non-credential based claims.